src/libclipper/src/containers.cpp

```diff
@@ -108,8 +138,17 @@ void ActiveContainers::add_container(VersionedModelId model, int connection_id,
                      model.get_name(), model.get_id(), connection_id,
                      replica_id, get_readable_input_type(input_type));
   boost::unique_lock<boost::shared_mutex> l{m_};
+
+  // Set a default batch size of 1
+  int batch_size = 1;
```

Comment:
The default value should be `-1` if the user has not defined a batch size for the container's associated model.